gradle/shared-with-buildSrc/mirrors.settings.gradle.kts

```
@@ -25,14 +25,14 @@ val originalUrls: Map<String, String> = mapOf(
     "mavencentral" to "https://repo.maven.apache.org/maven2/",
     "google" to "https://dl.google.com/dl/android/maven2/",
     "gradle" to "https://repo.gradle.org/gradle/repo",
-    "gradleplugins" to "https://plugins.gradle.org/m2",
+    "gradle-prod-plugins" to "https://plugins.gradle.org/m2",
```

```suggestion
    "gradle-plugins" to "https://plugins.gradle.org/m2",
```

Why add `prod` here? The key should describe the actual repository, and `plugins.gradle.org` is the Gradle plugins repository, right?

---

There's also a `gradle-dev-plugins` mapping in the env var. That's why I added `prod` to make that consistent. Should we keep `gradle-dev-plugins` and `gradle-plugins` instead?

---

Ah, I didn't see the `gradle-dev-plugins` mapping. Then `gradle-prod-plugins` is fine.